domenetjenester/brevbestiller/src/test/java/no/nav/foreldrepenger/melding/brevmapper/brev/VarselOmRevurderingDokumentdataMapperTest.java

```diff
@@ -0,0 +1,196 @@
+        assertThat(varselOmRevurderingDokumentdata.getAdvarselKode()).isEqualTo(RevurderingVarslingÅrsak.ARBEIDS_I_STØNADSPERIODEN.getKode());
+        assertThat(varselOmRevurderingDokumentdata.isFlereOpplysninger()).isFalse();
+    }
+
+    @Test
+    public void skal_mappe_felter_for_brev_til_verge() {
+        // Arrange
+        Behandling behandling = opprettBehandling();
+        DokumentFelles dokumentFelles = lagDokumentFelles(true);
+        DokumentHendelse dokumentHendelse = lagDokumentHendelse(RevurderingVarslingÅrsak.ARBEIDS_I_STØNADSPERIODEN);
+
+        // Act
+        VarselOmRevurderingDokumentdata varselOmRevurderingDokumentdata = dokumentdataMapper.mapTilDokumentdata(dokumentFelles, dokumentHendelse, behandling);
+
+        // Assert
+        assertThat(varselOmRevurderingDokumentdata.getFelles()).isNotNull();
+        assertThat(varselOmRevurderingDokumentdata.getFelles().getSøkerNavn()).isEqualTo(SØKERS_NAVN);
+        assertThat(varselOmRevurderingDokumentdata.getFelles().getSøkerPersonnummer()).isEqualTo(formaterPersonnummer(SØKERS_FNR));
+        assertThat(varselOmRevurderingDokumentdata.getFelles().getMottakerNavn()).isEqualTo(VERGES_NAVN);
+        assertThat(varselOmRevurderingDokumentdata.getFelles().getHarVerge()).isEqualTo(true);
+        assertThat(varselOmRevurderingDokumentdata.getFelles().getErKopi()).isEqualTo(false);
+    }
+
+    @Test
+    public void skal_gi_flere_opplysninger_når_ikke_JOBBFULLTID_er_årasak() {
+        // Arrange
+        Behandling behandling = opprettBehandling();
+        DokumentFelles dokumentFelles = lagDokumentFelles(false);
+        DokumentHendelse dokumentHendelse = lagDokumentHendelse(RevurderingVarslingÅrsak.ARBEID_I_UTLANDET);
+
+        // Act
+        VarselOmRevurderingDokumentdata varselOmRevurderingDokumentdata = dokumentdataMapper.mapTilDokumentdata(dokumentFelles, dokumentHendelse, behandling);
+
+        // Assert
+        assertThat(varselOmRevurderingDokumentdata.isFlereOpplysninger()).isTrue();
+    }
+
+    private FamilieHendelse opprettFamiliehendelse() {
+        FamilieHendelse.OptionalDatoer optionalDatoer = new FamilieHendelse.OptionalDatoer(Optional.empty(), Optional.of(TERMINDATO), Optional.empty(), Optional.empty());
+        return new FamilieHendelse(BigInteger.valueOf(ANTALL_BARN), false, false, FamilieHendelseType.TERMIN, optionalDatoer);
+    }
+
+    private Behandling opprettBehandling() {
+        return Behandling.builder()
+                .medId(1L)
+                .medBehandlingType(BehandlingType.REVURDERING)
+                .medBehandlingÅrsaker(of(BehandlingÅrsak.builder().medBehandlingÅrsakType(BehandlingÅrsakType.RE_ENDRING_FRA_BRUKER).build()))
+                .medBehandlingsresultat(Behandlingsresultat.builder()
+                        .medBehandlingResultatType(BehandlingResultatType.INNVILGET).build())
+                .build();
+    }
+
+    private DokumentFelles lagDokumentFelles(boolean tilVerge) {
```

Comment:
Kan du ikke bruke DatamapperTestUtil sin lagStandardDokumentFelles?

Reply:
Denne er nå tatt i bruk for alle 4 Dokgen-mapperne